ruby/benchmarks/bm_app_answer.rb

```diff
@@ -8,8 +13,12 @@ def ack(m, n)
   end
 end
 
-def the_answer_to_life_the_universe_and_everything
-  (ack(3,7).to_s.split(//).inject(0){|s,x| s+x.to_i}.to_s + "2" ).to_i
+def the_answer_to_life_the_universe_and_everything(m, n)
+  (ack(m, n)
+    .to_s
+    .split(//)
+    .inject(0) { |s, x| s + x.to_i }
+    .to_s + "2").to_i
 end
 
-answer = the_answer_to_life_the_universe_and_everything
+answer = the_answer_to_life_the_universe_and_everything(3, 7) # => 42
```

Comment:
+1. 

This sounds like something that can be benchmarked for.. :stuck_out_tongue: